concord_cli/config.py

```diff
@@ -73,26 +64,36 @@ def write_defaults(new_defaults, current, writepath):
     with open(writepath , 'w') as outfile:
         json.dump(current, outfile)
 
-def init(new_defaults):
-    filepath = find_config(os.getcwd())
-    if filepath is not None:
-        print '%s already exists in %s' \
-            % (CONFIG_FILENAME, os.path.realpath(filepath))
+def init(location, new_defaults=None):
+    """ Creates config_file in location, asking user to overwrite if already
+    exists. Then prompts user for default values and writes to disk if -p
+    option has not been used"""
+    def duplicate_prompt(config_data, fpath):
+        print 'config_file already exits in', fpath
         inpt = raw_input('Overwrite? [y|n] ')
-        if inpt != "y":
-            print 'Exiting'
-            return
-    else:
-        filepath = os.path.join(os.getcwd(), CONFIG_FILENAME)
+        return fpath if inpt == "y" else sys.exit(1)
+
+    default_path = os.path.join(os.path.expanduser(location), CONFIG_FILENAME)
+    filepath = fetch_config(os.getcwd(), duplicate_prompt, default_path)
+
+    if new_defaults is None:
+        def prompt_default(k, v):
+            inpt = raw_input('-- for key "%s" value will be ["%s"]: ' % (k,v))
+            return v if inpt == "" else inpt
+
+        print '\nCreating %s, select or overwrite default values: ' % filepath
```

Comment:
OHHHHHHHH, that's interesting. you can't use a logger because of the formatting. Good thinking.